### PR TITLE
Add script to check SDL does not resend OnSeekMediaClockTimer without…

### PR DESCRIPTION
**Pull request by OVikhrov**
… hours parameter
@aarotar Please review

### CODE REVIEW

File: test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_hours_Absent.lua [hunk 1]
```diff
@@ -0,0 +1,96 @@
+--------------------------------------------------------------------------------------------
+-- Requirement summary:
```
Comment (aarotar) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_hours_Absent.lua]: @OVikhrov, please update Requirement summary with related functional requirement also

File: test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_hours_Absent.lua [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+-- [MOBILE_API] New OnSeekMediaClockTimer notification
+-- [HMI_API] New OnSeekMediaClockTimer notification
+-- 
+--
```
Comment (aarotar) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_hours_Absent.lua]: @OVikhrov, plase avoid redundant empty lines

File: test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_hours_Absent.lua (hunk 1 above)
Comment (OVikhrov) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_hours_Absent.lua]: Accepted

File: test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_hours_Absent.lua (hunk 2 above)
Comment (OVikhrov) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_hours_Absent.lua]: Accepted

**Comment by OVikhrov** [comment]
@aarotar Plese review iteration#2

**Pull request merged**